tools/skiplist/create_skipfile.py

```diff
@@ -0,0 +1,54 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+"""
+Converts Gerrit Review changed files list to CodeChecker skipfile.
+"""
+
+import argparse
+import json
+import re
+
+
+def create_skipfile(files_changed, skipfile):
```

Comment:
Inside the common library there is already a similar function to get the list of changed files:
https://github.com/Ericsson/codechecker/blob/0be059eba2c951288bbaec2b53ca285b2d717b75/codechecker_common/output/gerrit.py#L107-L138
Because you created a module in this PR I recommend you to move this function under this tool so we can use this in two places and we can avoid code duplication:
- you can use this __get_changed_files function to create skip file in this tool.
- and you can use this function in the common module.